app/scripts/controllers/meeting.status.ctrl.js

```diff
@@ -213,15 +213,47 @@ angular.module('dashboard')
         }
 
         function motionUpdated(aEvent) {
-            if (angular.isObject(aEvent)) {
+            if (angular.isObject(aEvent) && angular.isObject(aEvent.motion)) {
                 $log.debug("meetingStatusCtrl.motionUpdated", arguments);
-                // todo: pending implementation
+                var data = StorageSrv.getKey(CONST.KEY.MOTION_DATA);
+                if (angular.isObject(data)) {
+                    if (angular.isArray(data.objects)) {
+                        var notFound = true;
```

Comment:
Minor comment: "not" named variables are a bit of an anti-pattern. Why not simply `found = false`  :)